Godeps/Godeps.json

```diff
@@ -0,0 +1,27 @@
+{
+	"ImportPath": "github.com/Clever/redshifter",
+	"GoVersion": "go1.3.3",
```

Comment:
My machine did not have go1.4 so godep defaulted to using 1.3.3. Fixed in dafa1dfc283fdc7137f5ef0ae0b456725f4bb5a6.